templates/patterns/navigation_list/navigation_list.ui_patterns.yml

```diff
@@ -0,0 +1,30 @@
+navigation_list:
+  label: "Navigation list"
+  description: "The navigation list defines the children that will be shown on the parent page (current page)."
+  fields:
+    title:
+      type: "text"
+      label: "Title"
+      description: "Title of the child."
+      preview: "Page title section (child A)"
+    href:
+      type: "text"
+      label: "Url"
+      description: "Url of the child."
+      preview: "http://link-a.com"
```

Comment:
Is this field the URL used to link the title of the navigation list? If so shall we call it `Title URL`/ `title_url`?